src/bin/stratisd.rs

```diff
@@ -40,28 +44,44 @@ use devicemapper::Device;
 
 use libstratis::engine::{Engine, SimEngine, StratEngine};
 use libstratis::stratis::{StratisError, StratisResult, VERSION};
+use libstratis::stratis::{alarm, buff_log};
 
 const STRATISD_PID_PATH: &str = "/var/run/stratisd.pid";
+const STRATISD_ALARM_SECONDS: u32 = 600;
+const DEFAULT_LOG_HOLD_MINUTES: i64 = 30;
 
 /// If writing a program error to stderr fails, panic.
 fn print_err(err: &StratisError) -> () {
     eprintln!("{}", err);
 }
 
+/// Print the engine state in a formatted way.
+fn print_engine_state(engine: &Engine) {
+    debug!("Engine state: \n{:#?}", engine);
+}
+
 /// Configure and initialize the logger.
 /// If debug is true, log at debug level. Otherwise read log configuration
 /// parameters from the environment if RUST_LOG is set. Otherwise, just
 /// accept the default configuration.
-fn initialize_log(debug: bool) -> () {
+fn initialize_log(debug: bool) -> buff_log::Handle<env_logger::Logger> {
     let mut builder = Builder::new();
+    builder.default_format_timestamp(false);
```

Comment:
Yes, it does that. This is to turn off the timestamp in the embedded log implementation, which is not useful since the time it sees the message is not the same as when it originated.